webrtc.html

```diff
@@ -1755,6 +1762,36 @@ <h4>Set the RTCSessionDescription</h4>
                               </li>
                             </ol>
                           </li>
+                          <li>
+                            <p>If <var>description</var> is of type <code>"answer"</code>
+                            or <code>"pranswer"</code>, then run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p>If the offer contained a request to send simulcast and
+                                the answer indicates that simulcast is not supported or
+                                desired by the answerer, then set the
+                                <code><a data-link-for="RTCRtpEncodingParameters">active</a></code>
+                                flag to <code>false</code> on all the <var>sendEncodings</var>
+                                except the first, in the <code><a>RTCRtpSender</a></code>
+                                and abort these sub steps.</p>
+                              </li>
+                              <li>
+                                <p>If the offer contained a request to send simulcast and
+                                the answer rejects any of the offered layers, then set the
+                                <code><a data-link-for="RTCRtpEncodingParameters">active</a></code>
+                                flag to <code>false</code> on the <var>sendEncodings</var>
+                                that correspond to rejected layers in the
+                                <code><a>RTCRtpSender</a></code>.</p>
+                              </li>
+                              <li>
+                                <p>Update the paused status of simulcast layers by setting the
+                                <code><a data-link-for="RTCRtpEncodingParameters">active</a></code>
+                                flag on the <var>sendEncodings</var> in the
```

Comment:
Setting it to what? I found no reference to "pause status" in this spec or in JSEP. Is there a section in JSEP we can reference?

Reply:
pause status is mentioned in [I-D.ietf-mmusic-sdp-simulcast](https://tools.ietf.org/html/draft-ietf-mmusic-sdp-simulcast-13) and that is referenced by jsep:

> When using JSEP to transmit multiple encodings from a RtpSender, the
>  techniques from [I-D.ietf-mmusic-sdp-simulcast] and
>  [I-D.ietf-mmusic-rid] are used.

There is somewhat of a missing flag to match this. This doesn't cause any issues, as the active flag can be used to indicate paused as well.
i will clarify that paused = !active.